dev/com.ibm.websphere.appserver.features/visibility/public/globalHandler-2.0/resources/l10n/io.openliberty.globalHandler-2.0.properties

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables Global Handler 2.0, \
+ which provides an API for global request and response handling including for both RESTful and XML Web Services.
```

Comment:
Replace with:  which provides an API for global request and response handling for both RESTful and XML Web Services. (Removed including)